app/controllers/my_controller.rb

```diff
@@ -128,9 +128,18 @@ def inbox
   end
 
   def reimbursements
-    @my_reports = current_user.reimbursement_reports
-    @my_reports = @my_reports.search(params[:q]) if params[:q].present?
-    @reports_to_review = Reimbursement::Report.submitted.where(event: current_user.events, reviewer_id: nil).or(current_user.assigned_reimbursement_reports.submitted)
+    case params[:filter]
+    when "created"
+      @my_reports = current_user.created_reimbursement_reports
+      @my_reports = @my_reports.search(params[:q]) if params[:q].present?
+    when "review"
+      @my_reports = Reimbursement::Report.submitted.where(event: current_user.events, reviewer_id: nil).or(current_user.assigned_reimbursement_reports.submitted)
+      @my_reports = @reports_to_review.search(params[:q]) if params[:q].present?
+    else
+      @my_reports = current_user.reimbursement_reports.concat(current_user.created_reimbursement_reports).concat(Reimbursement::Report.submitted.where(event: current_user.events, reviewer_id: nil).or(current_user.assigned_reimbursement_reports.submitted))
+      @my_reports = @reports_to_review.search(params[:q]) if params[:q].present?
```

Comment:
Noticed you had a typo with the instance variable name here. Shouldn't we just be calling it `@reports` now since we are combining `@my_reports` and `@reports_to_review`?
```suggestion
      @my_reports = @my_reports.search(params[:q]) if params[:q].present?
```